### DISCUSSION TITLE
How do I find an alias for a given domain?

### DISCUSSION THREAD

**Discussion by cnkcb** [simple-login/app#1951] [Q&A]
Love SL!  Have many, many aliases, but I can't remember what alias I used for a given domain.  Like, what was that address I used on Redfin.com?   That's just an example. I know I could go searching through old emails - whatever.  Except, the one I'm looking for hasn't been used in a couple years, and all those old emails are gone.....  So, hmm.  How to find it?

**Comment by 1ibno** [comment]
I had the same question! ...until I realized the search function is located behind the 'Show Filters' button in the upper right part of the UI (see screenshots).
![firefox_UFx1ALWNdh](https://github.com/simple-login/app/assets/94773681/0cb3d86d-6ec6-4746-af23-78ac58094b47)
![firefox_Z34pvRk5I0](https://github.com/simple-login/app/assets/94773681/88e7855d-7a5c-4d1b-8c78-28e0d0d9a81b)
